addons/stock/static/src/views/stock_orderpoint_list_controller.js

```diff
@@ -30,8 +30,10 @@ export class StockOrderpointListController extends ListController {
         this.actionService.doAction('stock.action_orderpoint_snooze', {
             additionalContext: { default_orderpoint_ids: resIds },
             onClose: () => {
-                this.actionService.doAction('stock.action_replenishment', {
-                    stackPosition: 'replaceCurrentAction',
+                this.actionService.doActionButton({
+                    type: 'object',
+                    name: 'action_view_orderpoints',
+                    resModel: 'product.product',
```

Comment:
You need to pass the product id so it knows which ones to show. To test, try creating 3+ reordering rule lines (i.e. in different locations) and only snooze 2 of them. It will show none of them afterwards. Depending on the context, you may also want to check if whether you should be calling it with `product.template` or `product.product`